service/chain/disk.go

```diff
@@ -17,23 +17,27 @@ package chain
 import (
 	"errors"
 	"fmt"
+	"math"
 
 	"github.com/dgraph-io/badger/v2"
+	"github.com/optakt/flow-dps/models/dps"
```

Comment:
Misplaced import

Reply:
ALREADY FIXED! No idea why it changed it O_o